_posts/en/newsletters/2020-09-09-newsletter.md

```diff
@@ -0,0 +1,196 @@
+      counterparty can broadcast a low-feerate version and then use a
+      [transaction pinning][topic transaction pinning] method to make it
+      impractical to fee bump.  If the attack is successful, it would
+      allow the attacker to spend the victim's money; if it was
+      unsuccessful, it might cost the attacker nothing extra if they
+      were already planning to send the transactions they used for
+      pinning (e.g. they were planning to consolidate UTXOs anyway).
+
+  Several potential solutions to these attacks were discussed.  The
+  [most promising solution][belcher collateral] was to require anyone
+  spending a coinswap UTXO to spend part of the value of one of their
+  other UTXOs to transaction fees.  This *collateral payment* would
+  eliminate the costless aspect of the previously described attacks.
+  Fully addressing the attacks might also require some other changes
+  related to timelocks, such as using a 1-block relative timelock ("`1
+  OP_CSV`") to prevent pinning.
+
+  In addition to the discussion helping to improve a privacy-enhancing
+  coinswap protocol, at least one participant [noted][zmnscpxj ln] that
+  some of the insights from the conversation might be applicable to
+  other protocols, such as LN.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Re-delegate absurd fee checking from mempool to clients][review club #19339] is
+a PR ([#19339][Bitcoin Core #19339]) by Gloria Zhao that proposes an updated
+version of [Bitcoin Core #15810][]. It adds separate fee-checking logic for
+clients that need to enforce max feerates and then removes the `nAbsurdFee`
+logic from `AcceptToMemoryPool`.
+
+The review club discussion covered general concepts before diving into the
+deeper technical aspects.
+
+{% include functions/details-list.md
+  q0="What does the `AcceptToMemoryPool` (ATMP) function in `validation.cpp` do?"
+  a0="ATMP is the function that Bitcoin Core calls to determine whether a transaction
+      is acceptable for inclusion in the mempool. In addition to
+      enforcing consensus rules, ATMP also applies the node's policy."
+  a0link="https://bitcoincore.reviews/19339#l-55"
+
+  q1="What is \"policy\"?"
+  a1="Policy is generally defined as each node's network transaction rules.  It excludes
+      consensus rules and the preferences of end-user clients (e.g. wallets)."
+  a1link="https://bitcoincore.reviews/19339#l-281"
+
+  q2="What is the absurd fee parameter, `nAbsurdFee`, used for?"
+  a2="`nAbsurdFee` is only used for locally submitted transactions, e.g. those
+      originating from the node's RPC interface and wallet. When a node receives
+      a transaction from the network (which is the majority of calls to ATMP),
+      it sends an empty value for `nAbsurdFee`."
```

Comment:
I think "sets" is better than "sends" here (since 'send' often implies being sent over a network)